packages/@ourworldindata/grapher/src/stackedCharts/MarimekkoChart.tsx

```diff
@@ -944,10 +888,50 @@ export class MarimekkoChart
                 />
             )
 
-        const { bounds, dualAxis } = this
+        const {
+            bounds,
+            dualAxis,
+            tooltipItem,
+            xColumn,
+            formatColumn: yColumn,
+            manager: { endTime, xOverrideTime },
+            inputTable: { timeColumn },
+            tooltipState: { target, position, fading },
+        } = this
+
+        const { entityName, entityColor, xPoint, bars } = tooltipItem ?? {}
+
+        const yValues =
+            bars?.map((bar: any) => {
+                const shouldShowYTimeNotice =
+                    bar.yPoint.value !== undefined &&
+                    bar.yPoint.time !== endTime
+
+                return {
+                    name: bar.seriesName,
+                    value: bar.yPoint.value,
+                    notice: shouldShowYTimeNotice ? bar.yPoint.time : undefined,
+                }
+            }) ?? []
+
+        // TODO: when we have proper time support to work across date/year variables then
```

Comment:
I know we recently revisited the way non-year granularity times are handled. Does that change anything about the logic here?

Reply:
I'm not quite sure what you're referring to. I know we chatted about it but I'm not aware that we implemented anything regarding non-year times. Maybe @danyx23 or @marcelgerber know more.

Reply:
There was a point where page-level CSS was intruding, so i switched from h1/h2 to divs for the title & subtitle